File: app/src/main/java/com/example/ubnt/fragviewapp/Item.java
package com.example.ubnt.fragviewapp;

public class Item {

    String Name;
    int Image;

    public Item(String Name, int Image)
    {
        this.Image = Image;
        this.Name = Name;
    }
    public String getName()
    {
        return Name;
    }
    public int getImage()
    {
        return Image;
    }

    @Override
    public String toString() {
        return Name;
    }

    @Override
    public boolean equals(Object obj) {
        return Name.equals(obj.toString());
    }
}

File: app/src/main/java/com/example/ubnt/fragviewapp/Item.kt
package com.example.ubnt.fragviewapp

class Item(Name: String, Image: Int) {

    var name: String
        internal set
    var image: Int = 0
        internal set

    init {
        this.image = Image
        this.name = Name
    }

    override fun toString(): String {
        return name
    }

    override fun equals(obj: Any?): Boolean {
        return name == obj!!.toString()
    }
}